src/App.tsx

```diff
@@ -17,18 +64,42 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                onSetFilter={(filt: string) => setFilter(filt)}
```

Comment:
```suggestion
                onSetFilter={setFilter}
```
in such cases additional wrapper function is redundant
fix all similar cases